webapp/static/common.css

```diff
@@ -9,7 +9,7 @@ body {
 }
```

Comment:
```suggestion
  padding: 16px;
}
```

Reply:
This wouldn't make any difference normally, unless the viewport is narrower than 1200px, in which case this will apply some padding on the sides. I assume this is your intention.